packages/terra-site/src/examples/dialog/DialogDefault.jsx

```diff
@@ -1,8 +1,17 @@
 import React from 'react';
-import Dialog from 'terra-dialog/src/Dialog';
+import Dialog from 'terra-dialog';
+
+const header = <span>Header Stuff</span>;
+
+const footer = <span>Footer Stuff</span>;
 
 const DialogDefault = () => (
-  <Dialog />
+  <div style={{ height: '200px', width: '350px', border: 'dashed' }}>
```

Comment:
I know we're iterating on this component so this is a placeholder - in the end, we probably want to beef up this example. Currently, I think the test example is a better representation of the look/functionality.

Reply:
@yuderekyu Can you elaborate on this a little? To me, the example already looks almost identical to the test. How would you recommend beefing it up?

Reply:
Sorry, I should have been more specific - we could add long text to display the wrapping/ overflow scrolling, otherwise it looks fine

Reply:
@yuderekyu Made an example with long content: https://github.com/cerner/terra-core/pull/1173/commits/dcaa6ec487b139b9d11769956f38a79df5147ea0